src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
Cool, didn't thought about that. The first drive for that change was to fix build, as pointing to existing package was breaking things (due to a breaking change, of course, that does not always happen). The second drive for that was to allow to test the package in current development, e.g. in your working directory you're trying with some beta version.

In our team I don't think we already tried using packages from appveyor, so if you think it could be easier to reference a version like that, yes, I can change that.